openquakeplatform/openquakeplatform/ript/views.py

```diff
@@ -69,9 +79,12 @@ def validate_nrml(request):
     try:
         nrml.read(xml_file)
     except etree.ParseError as exc:
-        return _make_response(error_msg=exc.message.message,
-                              error_line=exc.message.lineno,
-                              valid=False)
+        if is_sendback:
+            pass
```

Comment:
Please add a comment about the fact that the button calling sendback_nrml is enabled only if the nrml is valid.